src/odemis/dataio/tiff.py

```diff
@@ -2359,9 +2368,14 @@ def _getOMEXML(self, tfile):
         tfile.SetDirectory(0)
         desc = tfile.GetField(T.TIFFTAG_IMAGEDESCRIPTION)
 
-        if (desc and ((desc.startswith(b"<?xml") and b"<ome " in desc.lower()) or
-                      desc[:4].lower() == b'<ome')):
+        if (desc and ((b"<?xml" in desc) and b"<ome " in desc.lower() or
+                desc[:4].lower() == b'<ome')):
```

Comment:
The last part (` desc[:4].lower()`) would deserve more love. It used to say: either start with xml + has ome, or start with ome. Now it says has xml or has ome or start with ome.

I think you could start by converting desc to lower(), and then checking for a line starting with xml/ome + it should contain ome.

You can do that with a single regex match, which you use later to find out the index.